src/engine.js

```diff
@@ -6,29 +6,36 @@ import dispatcher from './utils/dispatcher'
 
 
 
-export default class Engine{
-  constructor(canvas){
+export default class Engine {
+  constructor(canvas) {
     this.canvas = canvas
     this.context = canvas.getContext('2d')
     // Level Generation, get all objects that will be in the game
     // obstacles, monsters, powerps, player, end
     this.game = {...(map(this.canvas, this.context))}
   }
 
-  start(){
+  start() {
     this.loop();
```

Comment:
Should probably set `run = true` here.

Reply:
ok

Reply:
In normal circumstances I would have done that. Since the loop is looking for a `=== false` flag.. I would say that it's unnecessary. I'll add it but there is no functionality that requires it.